client/me/billing-history/utils.jsx

```diff
@@ -50,3 +52,26 @@ export const groupDomainProducts = ( originalItems, translate ) => {
 		} ),
 	];
 };
+
+export function renderTransactionAmount( transaction, { translate, addingTax = false } ) {
+	if ( ! config.isEnabled( 'show-tax' ) || ! transaction.tax_amount ) {
+		return transaction.amount;
+	}
+
+	const taxAmount = addingTax
+		? translate( '(+%(taxAmount)s tax)', {
+				args: { taxAmount: transaction.tax_amount },
+				comment: 'taxAmount is a localized price, like $12.34',
+		  } )
+		: translate( '(includes %(taxAmount)s tax)', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '+%(taxAmount)s tax' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).